dangerfile-ios.js

```diff
@@ -90,22 +90,5 @@ if (!hasIosFile && danger.git.deleted_files) {
 }
 
 if(hasIosFile){
-  console.log('hasIosFile');
-}
-// console.log('-----------------------------hasIosFile-----------------------------:'+hasIosFile);
-// if(hasIosFile){
-//   var runTestCmd='source ~/.bash_profile; '
-//     +'xcodebuild -project ios/sdk/WeexSDK.xcodeproj test '
-//     +'-scheme WeexSDKTests CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO '
-//     +'-destination "platform=iOS Simulator,name=iPhone 6"'
-//   runSuccess = shell.exec(runTestCmd,{ async: false, timeout: 8 * 60 * 1000, maxBuffer: 200 * 1024 * 1024 }).code == 0;
-//   if(!runSuccess){
-//     fail("ios platform run unit test failed!");
-//   }
-// }else{
-//   console.log('has no ios file changed.');
-//   message('has no ios file changed.');
-// }
-// message('ios test finished.')
-
-
+  console.log('hasIosFile!!!');
```

Comment:
You may need to consider rewriting the code with `if(hasIosFile)` block

Reply:
I'm sorry to forget add `if(hasIosFile)` block,
I will update the code

Reply:
the ` console.log('hasIosFile!!!');` was in `if(hasIosFile)` block.
for `dangerfile-ios.js` I just delete the commented code.

Reply:
Ok, my mistake